includes/class-wc-cart-totals.php

```diff
@@ -534,35 +539,49 @@ protected function calculate_item_subtotals() {
 	 */
 	protected function calculate_item_discounts() {
```

Comment:
@mikejolley I'm stuck on this section. I have all of the discounts showing up in the cart with the correct discount amounts, but not the cart totals. The issue is that I'm not sure how to correctly change the cart total + tax total to account for discounts. Any help/advice you have would be appreciated.